tests/conftest.py

```diff
@@ -3355,7 +3351,7 @@ def empty_base_data_context_in_cloud_mode_custom_base_url(
 ) -> CloudDataContext:
     project_path = tmp_path / "empty_data_context"
     project_path.mkdir()
-    project_path = str(project_path)
+    project_path = str(project_path)  # type: ignore[assignment]
```

Comment:
Non-blocking since this isn't actively changing anything, but changing the type of `project_path` feels bad. I think the best solution here would be just to update line 3362 to be `context_root_dir=str(project_path)`, since it looks like the string version is only used there.

Reply:
I want to followup with a PR to actually fix the typing issues in `conftest.py` since it's a core file that almost every other test relies on.

I'm trying to get this one in quickly since new v1 related PR's that merge into develop will almost certainly require changes to the exclude list or the existing type-ignores.